java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -467,19 +467,27 @@ private static Object[] geospatialCommands(BaseTransaction<?> transaction) {
     }
 
     private static Object[] bitmapCommands(BaseTransaction<?> transaction) {
-        String key = "{key-" + UUID.randomUUID();
+        String key1 = "{key}-1" + UUID.randomUUID();
+        String key2 = "{key}-2" + UUID.randomUUID();
```

Comment:
nit - it will be easier to debug if needed
```suggestion
        String key1 = "{bitmapKey}-1" + UUID.randomUUID();
        String key2 = "{bitmapKey}-2" + UUID.randomUUID();
```